lib/ui/card.tsx

```diff
@@ -36,32 +47,73 @@ const CardTitle = React.forwardRef<
 >(({ className, ...props }, ref) => (
   <h3
     ref={ref}
-    className={cn(
-      "text-2xl font-semibold leading-none tracking-tight",
-      className
-    )}
+    className={cn("text-base font-semibold", className)}
     {...props}
   />
 ))
 CardTitle.displayName = "CardTitle"
 
-const CardDescription = React.forwardRef<
+const CardSubtitle = React.forwardRef<
   HTMLParagraphElement,
-  React.HTMLAttributes<HTMLParagraphElement>
+  React.HTMLAttributes<HTMLHeadingElement>
 >(({ className, ...props }, ref) => (
-  <p
+  <h3
     ref={ref}
-    className={cn("text-sm text-muted-foreground", className)}
+    className={cn("text-base font-normal text-muted-foreground", className)}
     {...props}
   />
 ))
-CardDescription.displayName = "CardDescription"
+CardSubtitle.displayName = "CardSubtitle"
+
+const CardInfo = React.forwardRef<
+  HTMLDivElement,
+  React.HTMLAttributes<HTMLDivElement>
+>(({ className, ...props }, ref) => (
+  <div
+    ref={ref}
+    className={cn("-ml-1 flex h-6 w-6 items-center justify-center", className)}
+  >
+    <TooltipProvider>
+      <Tooltip>
+        <TooltipTrigger
+          className="h-5 w-5 cursor-help text-muted-foreground"
+          title="Info"
+        >
+          <Icon icon={InfoCircleLine} size="md" />
+        </TooltipTrigger>
+        <TooltipContent>
+          <p {...props} />
```

Comment:
This is a bit weird. Shouldn't we just receive the contents as a string and put it as children of `<p>`?